judge/forms.py

```diff
@@ -284,3 +285,21 @@ def clean_key(self):
         if Contest.objects.filter(key=key).exists():
             raise ValidationError(_('Contest with key already exists.'))
         return key
+
+
+class ProblemPointsVoteForm(ModelForm):
+    class Meta:
+        model = ProblemPointsVote
+        fields = ['points', 'note']
+        # The default 'required' error message is triggered if non-integer data is sent
+        # even if the user enters data into the field, due to the form serialization replacing
+        # the invalid data with an empty string. It is overridden here to provide a suitable
+        # response, along with the other errors for consistency.
+        error_messages = {
+            'points': {
+                'required': _('Please enter an integer.'),
+                'min_value': _('Please enter an integer larger than or equal to %s.') % settings.DMOJ_PROBLEM_MIN_USER_POINTS_VOTE,
+                'max_value': _('Please enter an integer less than or equal to %s.') % settings.DMOJ_PROBLEM_MAX_USER_POINTS_VOTE,
+                'invalid': _('Please enter a whole number.')
```

Comment:
Could we use the `Please enter an integer.` string again instead of having a different string that is identical in meaning?

Reply:
Honestly I'm not even really that fond of the error messages, we only really need `required`. What do you think?

Reply:
I've removed these overrides, except for `required`.